nibabel/spatialimages.py

```diff
@@ -516,11 +516,11 @@ def __str__(self):
         shape = self.shape
         affine = self.affine
         return '\n'.join((str(self.__class__),
-                          'data shape %s' % (shape,),
+                          f'data shape {shape}',
                           'affine: ',
-                          '%s' % affine,
+                          f'{affine}',
                           'metadata:',
-                          '%s' % self._header))
+                          f'{self._header}'))
```

Comment:
Same here.

```suggestion
                          str(self._header)))
```

Or `'metadata:\n{self._header}'))`.